### ISSUE TITLE
package itself brakes adding files 

### ISSUE THREAD

**Issue by sanyatuning** [heartsucker/node-deb#25]
This line change to working dir so adding files or dirs will not work in node-deb.
https://github.com/heartsucker/node-deb/blob/develop/templates/executable#L6

**Comment by heartsucker** [comment]
Hey @sanyatuning, thanks for opening this issue. I don't think I fully understand what the problem is. Using `cd` prevents us from adding files? Could you elaborate please.

**Comment by sanyatuning** [comment]
We use node-deb installed by deb package.

So when I try to create a package:
```
$ cd ~/project
$ node-deb -- dist/ node_modules/ package.json
Failure: File does not exist: 'dist/'. Aborting
```
Becouse node-deb change the working directory to /usr/local/node-deb/app.

**Comment by heartsucker** [comment]
Fixed by commit a3f7f148911c6aed3c8e14b2015bbbce32c3d0fc (a while ago)
